README.md

```diff
@@ -394,7 +394,7 @@ Enable the option `#define DISPLAY_ROTATE_180_DEGREES` in `config.h`. This shoul
 
 Edit the file `config.h` in a text editor (a command line one such as `pico`, `vim`, `nano`, or SSH map the drive to your host), and find the appropriate line in the file. Add comment lines `//` in front of that text to disable the option, or remove the `//` characters to enable it.
 
-After having edited and saved the file, reissue `make -j` in the build directory and restart fbcp-ili9341.
+After having edited and saved the file, reissue ``make -j `nproc` `` in the build directory and restart fbcp-ili9341.

```

Comment:
Hmm, is `make -j` not enough to get to parallel compilation? Or does that not work in some cases?

Reply:
To be honest I don't really know if -j is a problem. The docs say

> If there is nothing looking like an integer after the ‘-j’ option, there is no limit on the number of job slots.

What effect that has I don't know, I'm just used to always using nproc.

Reply:
Ok, let's go with just `make -j` then, no need to specify the `nproc`.